examples/demo/pingpong/src/pingpong/j/model/package-info.java

```diff
@@ -1,4 +1,4 @@
-@Model
+@Model("PingPong")
```

Comment:
If supported, please update _XPingPong_ as well to keep it consistent with _JPingPong_. What about _Clock_?

Reply:
Colock contains more not supported elements so I would postpone to support it.